docs/content/docs/concepts/workflow/hooks/git-repo-webhook.md

```diff
@@ -12,3 +12,5 @@ You have to:
 * add a Repository Webhook on the root pipeline, this pipeline have the application linked in the [context]({{< relref "/docs/concepts/workflow/pipeline-context.md" >}})
 
 GitHub / Bitbucket / GitLab are supported by CDS.
+
+> When you add a repository webhook it will also automatically delete your runs which are linked to a deleted branch (24h after branch deletion).
```

Comment:
```suggestion
> When you add a repository webhook, it will also automatically delete your runs which are linked to a deleted branch (24h after branch deletion).
```